checkers_db.go

```diff
@@ -33,12 +33,14 @@ func GetCheckersInfo() []*CheckerInfo {
 // NewChecker returns initialized checker identified by an info.
 // info must be non-nil.
 // Panics if info describes a checker that was not properly registered.
-func NewChecker(ctx *Context, info *CheckerInfo) *Checker {
+//
+// params argument specifies per-checker options.NewChecker. Can be nil.
+func NewChecker(ctx *Context, info *CheckerInfo, params map[string]interface{}) *Checker {
```

Comment:
`params parameters` or leave it 'type weak' for the users?

Reply:
The type is unexported and users are unlikely to need any methods that `parameters` provide.

I think of `parameters` type as a convenience wrapper for internal use, for the application that integrates checkers, it's just a key-value config.